src/plugins/curlget/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 if (DEPENDENCY_PHASE)
 	find_package(CURL)
-
 	if(NOT CURL_FOUND)
 		remove_plugin(curlget "Curl-dev not found")
 	endif ()
+	find_package(OpenSSL)
```

Comment:
what if OpenSSL was not found?

Why is it needed? Why use md5?

To fix the OpenSSL problem you need to pack all arguments you need later together here.
E.g. set (CURLGET_INCLUDE_DIRS ${CURL_INCLUDE_DIR}	${OPENSSL_INCLUDE_DIR})